components/config/caching.rst

```diff
@@ -38,7 +38,7 @@ should be regenerated::
         $resources = array();
 
         foreach ($yamlUserFiles as $yamlUserFile) {
-            // see the previous article "Loading resources" to
+            // see the next article "Loading resources" to
```

Comment:
Actually, it isn't the very next article neither (which is "Defining and Processing Configuration Values"). So I think we can better reword the comment by simply removing "previous". What do you think?